sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -84,6 +85,33 @@ private[sbtplugin] object ScalaJSPluginInternal {
     }
   }
 
+  private object FailedToStartCmd {
+    def unapply(t: Throwable): Option[String] = {
+        val causeChain = Iterator
+          .iterate(t)(_.getCause())
+          .takeWhile(_ != null)
+
+        causeChain.collectFirst {
+          case ExternalJSRun.FailedToStartException(cmd :: Nil, _) =>
```

Comment:
I'm pretty sure that this should at least be
```suggestion
          case ExternalJSRun.FailedToStartException(cmd :: _, _) =>
```
Otherwise it cannot match anything where there are arguments to the process.